code/game/turfs/simulated/walls.dm

```diff
@@ -182,7 +182,10 @@
 		return
 
 /turf/simulated/wall/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	user.delayNextAttack(8)
+	if(istype(W, /obj/item/tool/crowbar/red)) // Mirroring original's behaviour
```

Comment:
what?

Reply:
nani

Reply:
Just plays that sound if it hits a wall rather than the generic pushing message.